manim/mobject/geometry.py

```diff
@@ -392,6 +392,28 @@ def __init__(self, start_point, end_point, **kwargs):
 
 
 class Circle(Arc):
+    """A circle.
+
+    Parameters
+    ----------
+    kwargs : Any
+        additional arguments to be passed to Arc
+
+    Examples
+    --------
+
+    .. manim:: CircleExample
+        :save_last_frame:
+
+        class CircleExample(Scene):
+            def construct(self):
+                circle = Circle(radius=1.0)
```

Comment:
Maybe increase the radius? There is a lot of free space in the scene

Reply:
I think it's worthwhile to show the default value for how the mobject looks like. It is a little barren, but I don't think that's too bad, especially since the shape is so simple

Reply:
That's convincing :slightly_smiling_face:

Reply:
You can add multiple Circles at once in the same frame. A better example would be to show various Circles with different parameters (use `.next_to`, or `.arrange` to sit them next to each other).